modules/identity-provider-github-about.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-github-identity-provider.adoc
+
+[id="identity-provider-github-about_{context}"]
+= About GitHub authentication
+
+Configuring GitHub authentication allows users to log in to {product-title} with their GitHub credentials. To prevent anyone with any GitHub user ID from logging in to your {product-title} cluster, you can restrict access to only those in specific GitHub organizations.
```

Comment:
Do we need/want  a link to GH here? Some of the other 3rd party authenticators have links (GitLab, Google, OpenID)

Reply:
Good question, though I searched around and couldn't find an obvious choice.

@stlaz @ibihim Can you help me with what would be the best place in the GitHub docs to link to? Would this one be appropriate? https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/connecting-with-third-party-applications

Reply:
This one's the correct section: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/authorizing-oauth-apps

I am not sure whether content is helpful though.